### DISCUSSION TITLE
none zero value of lift for 2d symmetric airfoil

### DISCUSSION THREAD

**Discussion by jjywow** [su2code/SU2#2030] [Q&A] [answered]
Hello, 

I have question about lift for 2d symmetric airfoil.

I conducted steady simulation withmesh_NACA0012_inv.su2 mesh file which is in following tutorial 
(Unconstrained shape design of a transonic inviscid airfoil at a cte)
su2code.github.io/tutorials/Inviscid_2D_Unconstrained_NACA0012/

I thought lift should be zero or nearly zero (due to numerical error caused by mesh).
But I got a lift value 0.001169. I think order of 1e-6 can be considered as numerical error but 1e-3 order is not negligible.

Is it normal? or can we see order of 1e-3 is neglegible?

Thank you very much for your attention

**Comment by pcarruscag** [comment, marked as answer]
It depends if the mesh is symmetric or not. The error should reduce by refining the mesh.

**Reply by jjywow** [reply]
Thanks for your answer. I'm trying to generate 2d O-type mesh for NACA64A010 with Gmsh. How can I generate good mesh? Is there some typical technique or tips? I tried to put circles at leading and tailing edges to apply finer mesh but I'm wondering if there is better technique...
